Sources/SwiftDocC/Model/Rendering/RenderContentCompiler.swift

```diff
@@ -206,11 +206,17 @@ struct RenderContentCompiler: MarkupVisitor {
     }
     
     mutating func resolveTopicReference(_ destination: String) -> ResolvedTopicReference? {
-        if let cached = context.referenceIndex[destination] {
+         if let cached = context.referenceIndex[destination] {
```

Comment:
Nit: Accidental space here?
```suggestion
        if let cached = context.referenceIndex[destination] {
```

Reply:
Yes, that was accidental.